src/Illuminate/Container/Container.php

```diff
@@ -182,8 +184,8 @@ public function when($concrete)
     public function bound($abstract)
     {
         return isset($this->bindings[$abstract]) ||
-               isset($this->instances[$abstract]) ||
-               $this->isAlias($abstract);
+            isset($this->instances[$abstract]) ||
+            $this->isAlias($abstract);
```

Comment:
Can you please undo all indentation changes in this PR? Keeps the diff clean to review.

Reply:
will do thanks, I thought I read somewhere that I should not bother too much because styleCI will fix it, but sure!